corehq/apps/hqwebapp/tests/test_requirejs.py

```diff
@@ -48,6 +50,7 @@ def _categorize_file(filename):
 
         cls._run_jobs(cls.js_files, _categorize_file)
 
+    @attr(slow=16)  # due to setup
```

Comment:
Would be nice if we could tag the setup method rather than a random test on the class with slow setup.

Reply:
Agreed. I'm not sure how easy that will be. Another thing I've run into is that since we skip a lot of tests in our divided tests, it's not clear where that time is getting allocated. May need to think about this more if it becomes an issue, for now I'm ok with commenting about it